dark_rabbit/models/outgoing_event.py

```diff
@@ -0,0 +1,43 @@
+from odoo import fields, models
+
+
+class DarkRabbitOutgoingEvent(models.Model):
+    _name = "dark.rabbit.outgoing.event"
+
+    body = fields.Text(required=True)
+
+    outgoing_event_type_id = fields.Many2one(
+        comodel_name="dark.rabbit.outgoing.event.type"
+    )
+
+    connection_id = fields.Many2one(
+        comodel_name="dark.rabbit.connection", required=True
+    )
+
+    exchange = fields.Char(required=True, index=True)
+
+    routing_key = fields.Char(required=True, index=True)
+
+    sent_at = fields.Datetime()
+
+    error = fields.Boolean(readonly=True)
+    error_msg = fields.Text(readonly=True)
+
+    created_at = fields.Datetime(string="Creation date", automatic=True, readonly=True)
+
+    def add(self, e_type, body):
+        event_type = self.env["dark.rabbit.outgoing.event.type"].search(
+            [("code", "=", e_type)]
+        )
```

Comment:
Use `sudo` here, because this method could be called outside by user who does not have access for event types